src/app/api/api_v1/endpoints/sequences.py

```diff
@@ -31,6 +32,37 @@ async def verify_org_rights(
         raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail="Access forbidden.")
 
 
+def _resolve_cone(azimuth: float, bboxes_str: str, aov: float) -> Tuple[float, float]:
```

Comment:
Hey @frgfm ,
As we discussed, selecting the bbox with the highest score is problematic in the case of fires that last a bit longer. As the fire grows, the bbox gets bigger, which reduces cone's precision 

Two options:

1. The best one in my opinion: we take the highest score, but only among the first 4 (the length of the sequence analyzed by the engine)

2. We simply take the first one

Reply:
This is taking the highest confidence bbox on the first detection of the sequence, not the highest conf bbox in the entire sequence :) Isn't that what we wanted?

Reply:
oh yes my bad I read it a bit too quickly 😄